data/human/human missions.txt

```diff
@@ -5733,3 +5733,227 @@ mission "Argosy Hijacking (Declined)"
+			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
+			choice
+				`	"Yes, my name's Captain <last>."`
+				`	"Depends on what you want."`
+					goto suspicious
+			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course."`
+				goto helpornot
+			label suspicious
+			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course.`
+			label helpornot
+			choice
+				`	"Of course I can."`
+					goto agreed
+				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
+			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
+				decline
+			label agreed
+			`	"Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We've been looking for a ride for some time now. We were starting to worry that we wouldn't get there in time! You'll have to bring us there quickly."`
+			choice
+				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
+					goto notime
+				`	"Yes, a lot of captains are quite busy these days."`
+			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
+				goto ship
+			label notime
+			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
+			label ship
+			`	You lead them to your ship, and help them with their bags as they board.`
+				accept
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`
+		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```

Comment:
```suggestion
		dialog `When you arrive at <destination>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```

Reply:
```suggestion
		dialog `When you arrive at <planet>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```
`<planet>` is better than `<destination>` in this instance.